packages/boxel-motion/addon/behaviors/spring.ts

```diff
@@ -247,7 +257,7 @@ export default class SpringBehavior implements Behavior {
     }
   }
 
-  private springToFrames(values: SpringValues): Frame[] {
+  private *springToFrames(values: SpringValues) {
     let { fromValue = 0, toValue = 1, initialVelocity = 0 } = values;
 
     if (fromValue === toValue && initialVelocity === 0) {
```

Comment:
Also here?

Reply:
I'll have to double check this one, maybe we did have this check (but down here instead of at the top). Having an initial velocity could mean an interpolation for a spring even if from is equal to to I think

Reply:
As we discussed I misunderstood your comment xD. We should indeed just `return;`. I've updated mainly the Spring code a bit since we were exiting too late there (after yielding the delay frames).

Reply:
Should be resolved